doc/rest-api.md

```diff
@@ -518,6 +518,26 @@ Input (using a local container):
                    "source": "my-old-container"}                                        # Name of the source container
     }
 
+Input (using a remote container, in push mode sent over the migration websocket via client proxying):
```

Comment:
There is an entry above this that say "Only "pull" is supported for now". This isn't true anymore and should be changed.